src/main/java/core/basesyntax/service/DataConverterImpl.java

```diff
@@ -0,0 +1,35 @@
+package core.basesyntax.service;
+
+import core.basesyntax.model.FruitTransaction;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class DataConverterImpl implements DataConverter {
+
+    public static final int TYPE = 0;
+    public static final int FRUIT = 1;
+    public static final int QUANTITY = 2;
+
+    @Override
+    public List<FruitTransaction> convertToTransaction(List<String> inputReport) {
+        List<FruitTransaction> transactions = new ArrayList<>();
+        for (String line : inputReport) {
+            String[] words = line.split(",");
```

Comment:
Consider using a more descriptive variable name than 'words' for the array resulting from the split operation. 'transactionData' or 'transactionParts' might be more informative.